### PR TITLE
Lang

### PR DESCRIPTION
**Pull request by glouw**

### CODE REVIEW

File: lang/test.lang [hunk 1]
```diff
@@ -0,0 +1,74 @@
+mario[] -> static
```
Comment (Kully) [inline lang/test.lang]: `test.asm` doesn't have a `main` label in it and the first line is `GOTO main`

File: lang/test.lang [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    else if(0)
+    {
+        ret = 1
+    }
+    else
+    {
+        ret = 2
+    }
+    return ret
+}
```
Comment (Kully) [inline lang/test.lang]: I think making an `EXIT` in `test.asm` is a good idea so the program counter doesn't spill over to the next file? WDYT?

File: lang/test.lang (hunk 1 above)
Comment (glouw) [reply lang/test.lang]: ye my bad there was no main() in test.lang

File: lang/test.lang (hunk 2 above)
Comment (glouw) [reply lang/test.lang]: ye last thing in main() should always be EXIT. C does the same thing

**Pull request closed without merging**